lib/IDE/SyntaxModel.cpp

```diff
@@ -418,6 +418,21 @@ CharSourceRange parameterNameRangeOfCallArg(const TupleExpr *TE,
   return CharSourceRange();
 }
 
+static CharSourceRange findDocCommentRange(DeclAttributes Attrs) {
+  for (auto Attr : Attrs) {
+    if (auto DocAttr = dyn_cast_or_null<RawDocCommentAttr>(Attr)) {
+      return DocAttr->getCommentRange();
```

Comment:
Well, interface printing uses [`Decl::getRawComment`](https://github.com/apple/swift/blob/7ee42994c85c3a3b61e75513aafa2c9962977bfa/lib/AST/RawComment.cpp#L129) for extracting doc comments, but I'm not sure we can use that here. It might be too costly for this purpose.
@nkcsgexi ?